nemo/collections/nlp/models/language_modeling/megatron_lm_encoder_decoder_model.py

```diff
@@ -707,7 +707,7 @@ def setup(self, stage=None):
         # when using pipeline model parallel the final stage need to initialize word embeddings
         if parallel_state.get_pipeline_model_parallel_world_size() > 1:
             self.enc_dec_model.sync_initial_word_embeddings()
-            if self.enc_dec_model.position_embedding_type != 'relative':
+            if self.cfg.position_embedding_type != 'relative':
```

Comment:
To make this backward compatibile to old models without `position_embedding_type`, you will need to use `self.cfg.get('position_embedding_type', 'learned_absolute')`.

Reply:
This is why CI is failing on the BART test right now.